webpack.config.js

```diff
@@ -117,6 +116,7 @@ jsLoader = {
 };
 
 if ( CALYPSO_ENV === 'development' ) {
+	const DashboardPlugin = require( 'webpack-dashboard/plugin' );
```

Comment:
<img width="494" alt="screen shot 2016-10-18 at 2 06 22 pm" src="https://cloud.githubusercontent.com/assets/1270189/19496889/81e93472-953e-11e6-9199-dd689e0ede78.png">

@dmsnell looks like I'm running into some issues with the webpack dashboard-plugin in the docker build. Looks like we'll need to move that require.

I think npm has updated it's install behavior with regards to the shrinkwrap. I now see differences between `npm install` vs `npm install --production` with a shrinkwrap that has dev dependencies.